src/main/webapp/course.html

```diff
@@ -222,7 +206,7 @@ <h1 class="p-4 text-center">Add Your Review</h1>
           </div>
         </form>
         <button
-          id="form-submit"
```

Comment:
probably keep this form-submit or change this to form-submit-button because Luke told me button was too vague

Reply:
Will change to form-submit :)